idb/postgres/postgres_migrations.go

```diff
@@ -23,22 +23,22 @@ import (
 	"github.com/algorand/indexer/types"
 )
 
-// rewardsMigrationIndex is the index of m5RewardsAndDatesPart2.
-const rewardsMigrationIndex = 5
+// rewardsMigrationIndex is the index of m6RewardsAndDatesPart2.
+const rewardsMigrationIndex = 6
 
 func init() {
 	migrations = []migrationStruct{
 		{m0fixupTxid, false, "Recompute the txid with corrected algorithm."},
 		{m1fixupBlockTime, true, "Adjust block time to UTC timezone."},
 		{m2apps, true, "Update DB Schema for Algorand application support."},
 		{m3acfgFix, false, "Recompute asset configurations with corrected merge function."},
```

Comment:
add a comment after m3acfgFix
`// Above are the migrations as of 2.2.1`
And after 2.3.1 we should add similar after m6
I could go back and look up the history of when even earlier things were released but it doesn't matter right now and this bit of accounting is repeatedly confusing to me about what we can and mustn't do.

Reply:
good idea, I'll add some comments